src/test/suite/fixtures/recorded/languages/java/takeTypeBlueLook.yml

```diff
@@ -0,0 +1,39 @@
+languageId: java
```

Comment:
Not worth rewriting all these tests, but I thought we had aligned on "clear" instead of "take" to make these tests easier to read. Makes it a bit painful having to manually count offsets to figure out what the test is doing

Reply:
You're completely correct.

Reply:
Fwiw I'm planning a rework of the tests to make modifier tests clearer / more compact, and then the "clear" / "take" thing won't matter.  Not sure if I've already filed an issue but I will at some point

Reply:
than that sounds really useful